extensions/adobe/experience/offer-management/selection-constraint.schema.json

```diff
@@ -0,0 +1,36 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$id":
+    "https://ns.adobe.com/xdm-extensions/adobe/experience/offer-management/selection-constraint",
+  "type": "object",
+  "title": "Offer Selection Constraint",
+  "description": "Offer Selection Constraint",
+  "definitions": {
+    "selection-constraint": {
+      "properties": {
+        "xdm:startDate": {
```

Comment:
Would it be possible to somehow indicate that these are date strings? This would be helpful for the XCORE UI components we're implementing.